zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -81,12 +83,25 @@ angular.module('zeppelinWebApp').controller('NotebookCtrl', function($scope, $ro
 
   //Export notebook
   $scope.exportNotebook = function() {
-    var jsonContent = 'data:text/json;charset=utf-8,' + JSON.stringify($scope.note);
-    var encodedUri = encodeURI(jsonContent);
-    var link = document.createElement('a');
-    link.setAttribute('href', encodedUri);
-    link.setAttribute('download', $scope.note.name + '.json');
-    link.click();
+    var jsonContent = JSON.stringify($scope.note);
+    if (browserDetectService.detectIE()) {
+      jQuery('h3').append('<iframe id="myFrame" style="display:none"></iframe>');
+      var myFrame = jQuery('#myFrame')[0].contentWindow;
+      myFrame.document.open('text/html', 'replace');
+      myFrame.document.write(jsonContent);
+      myFrame.document.close();
+      myFrame.focus();
+      myFrame.document.execCommand('SaveAs', true, $scope.note.name + '.json');
+      jQuery('h3 iframe').remove();
+    } else {
+      jsonContent = 'data:image/svg;charset=utf-8,' + encodeURIComponent(jsonContent);
+      jQuery('h3').append('<a class="exportNotebook"></a>');
+      jQuery('h3 a.exportNotebook').attr('href', jsonContent);
```

Comment:
This could be refactored to:
```
var exportElement = angular.element('h3 a.exportNotebook')
exportElement.attr('href', jsonContent);
exportElement.attr('download', $scope.note.name + '.json');
...
```